### ISSUE TITLE
Add a rule against one-hit kills in RP scenarios

### ISSUE THREAD

**Issue by katojishifu** [MrFrostt/suggestions#24]
This includes nerfing guns, so you can no longer surprise 1 hit kill people when sneaking. It forces more roleplay.

**Comment by katojishifu** [comment]
Yeah, I understand the fact you can litterally just shoot someone in the head, and then they're dead, right?

But the thing is, it offers little to no interaction other than, /me sneaks behind guy, grabs jaw and snaps neck instakill.* or /me sneaks behind guy, firing readied pistol at them to a headshot, instantly killing them*

It's just not fun to be instakilled like that, it offers no real thing other than, "Oh it's just NLR, I lost a few scripts" or something. Since you can't really resist a bullet, there's no risk factor involved for anyone involved.

**Comment by MrFrostt** [comment]
Yes, I can agree to that, hopefully, we can find consensus on that issue!
